animations/index.md

```diff
@@ -1,159 +1,88 @@
 
-* [Animations: Technical Overview](/animations/overview.html)<br>
-A look at some of the major classes in the animations library,
-and Flutter's animation architecture.
+With only a few lines of code, you can create attractive and effective animations using the implicit and transition widgets. And if you want to add complex animations, Flutter also provides  explicit animation widgets for customized effects so you can build your own animations using the `AnimatedBuilder` or `TransitionBuilder` widgets.
 
-* [Animation and Motion Widgets](/widgets/animation/)<br>
-A catalog of some of the animation widgets provided in the Flutter APIs.
+## Flutter animation widgets
+All Flutter animations include these basic properties:
+1. Animations must be associated with an object to animate.
+2. Animations must define what type of animation will be performed.
+3. Animations must define how long the animation will last.
 
-{% comment %}
-Until the landing page for the animation library is reworked, leave this
-link out.
-* The [animation
-library](https://docs.flutter.io/flutter/animation/animation-library.html)
-in the [Flutter API documentation](https://docs.flutter.io/)<br>
-The animation API for the Flutter framework.
-{% endcomment %}
+The Flutter SDK animation widgets form the basis for rendering your animation on iOS and Android platforms. Flutter animations widgets are *implicit*, *transition*, or *explicit* animations. If you’re starting out in animations, or you want to quickly add an animation to your app, try out the implicit or transition animation widgets.
 
-<hr>
+![Flutter animations](images/code_complexity3.png)
+<br>
+### Implicit animation widgets
 
-If there is specific animation documentation you'd like to see, please
-[file an issue](https://github.com/flutter/flutter/issues).
+ *Implicit animations* are animations that are already programmed or styled within pre-existing constraints—all you need to do is add the animated object, and set some basic parameters such as the duration. You don’t need to write the animation code to add listeners or tickers for example, and you don't need to tell the app when to start—the animation simply changes from the old to the new value.  
+ For more information, see [Implicit animations](/animations/implicit_widgets.html).
 
+
+### Transition animation widgets
+
+*Transition animations* implement the abstract class `AnimatedWidget`. Using the `AnimatedWidget` class allows you to separate the widget code from the animation code in the `setState()` call and it doesn’t need to maintain a `State` object to hold the animation.  Transition animations are similar to implicit animations because many of the animation elements are predefined for you. Use the  `AnimatedWidget` class (instead of `addListener()` and `setState()`) to create a reusable transition animation widget.  
```

Comment:
Another place where an example could convey this more succinctly. The InkWell's splash, for example.